diff --color -rau nanoflann-1.4.2/CMakeLists.txt nanoflann/nanoflann-1.4.2-source/CMakeLists.txt
--- nanoflann-1.4.2/CMakeLists.txt    2022-01-11 04:29:01
+++ nanoflann-1.4.2-source/CMakeLists.txt    2024-05-30 15:22:52
@@ -22,9 +22,14 @@
 
 # Enable a high level of warnings.
 if (CMAKE_COMPILER_IS_GNUCXX)
-	# The -Wno-long-long is required in 64bit systems when including sytem headers.
+        # The -Wno-long-long is required in 64bit systems when including sytem headers.
 	# The -Wno-variadic-macros was needed for Eigen3, StdVector.h
-	add_compile_options(-Wall -Wno-long-long -Wno-variadic-macros -O2 -mtune=native)
+	if (NOT APPLE)
+          add_compile_options(-Wall -Wno-long-long -Wno-variadic-macros -O2 -mtune=native)
+        endif()
+        if (APPLE) 
+          add_compile_options(-Wall -Wno-long-long -Wno-variadic-macros -O2)
+        endif()
 	# Workaround: Eigen <3.4 produces *tons* of warnings in GCC >=6. See http://eigen.tuxfamily.org/bz/show_bug.cgi?id=1221
 	if (NOT ${CMAKE_CXX_COMPILER_VERSION} LESS "6.0")
 		add_compile_options(-Wno-ignored-attributes -Wno-int-in-bool-context)
